com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/HDShadow.hlsl

```diff
@@ -94,7 +94,7 @@ float GetRectAreaShadowAttenuation(HDShadowContext shadowContext, float2 positio
 {
     // We need to disable the scalarization here on xbox due to bad code generated by FXC for the eye shader.
     // This shouldn't have an enormous impact since with Area lights we are already exploded in VGPR by this point.
-#if !defined(SHADER_API_XBOXONE) && (defined(PLATFORM_SUPPORTS_WAVE_INTRINSICS) && !defined(LIGHTLOOP_DISABLE_TILE_AND_CLUSTER))
+#if !defined(SHADER_API_XBOXONE) && !defined(SHADER_API_GAMECORE) && (defined(PLATFORM_SUPPORTS_WAVE_INTRINSICS) && !defined(LIGHTLOOP_DISABLE_TILE_AND_CLUSTER))
```

Comment:
isn't this working on Xbox? was guessing only gamecore is a problem right?

Reply:
This is an old problem with just the shadow data scalarization, FXC fails hard with the eye shader here.

Reply:
If it is a fix for xbox, then we should do a separate PR, it shouldn't be part of the GameCore PR. But yes, as we backport up to 2020.2 it is ok, eye shader wasn't present before

Reply:
we should add a comment for the shader data scalarezitation for eye sahder, so we don't forget it is because of this.

Reply:
It was there before and there is a comment up above :)

Reply:
```
    // We need to disable the scalarization here on xbox due to bad code generated by FXC for the eye shader.
    // This shouldn't have an enormous impact since with Area lights we are already exploded in VGPR by this point.

```